seed/ts-sdk/alias-extends/reference.md

```diff
@@ -31,7 +31,7 @@ await client.extendedInlineRequestBody({
 <dl>
 <dd>
 
-**request:** `SeedAliasExtends.InlinedChildRequest` 
+**request:** `InlinedChildRequest` 
```

Comment:
The user will import the type from the `SeedAliasExtends` namespace, so in _reference.md_, we need to continue to use the namespace import.